apps/site/pages/docs/api-extensions.mdx

```diff
@@ -219,4 +219,99 @@ query {
 }
 ```
 
+---
+
+## Consume new fields from pages
```

Comment:
What is the purpose of this section? Is it to **tell** our customers how to consume new fields on pages or is it to explain why this is a good idea or why they want to do it?

Reading it seems like we're trying to convince them to use GraphQL and given them motivation to use API Extension. If it is the case I'd suggest we rewrite this. The motivation should be already clear by this point on the docs.

Reply:
It was based on the introduction of this section in the RFC, but you are right, at this point of documentation is not necessary introduce the Graphql and fragments concept, I will just remove this first section.